spec/Cabin/Seat.vspec

```diff
@@ -70,6 +82,25 @@ MassageLevel:
   unit: percent
   description: Seat massage level. 0 = off. 100 = max massage.
 
+MassageStatus:
+  datatype: string
+  type: sensor
+  allowed: ['ON', 'OFF']
+  description: Massage status.  
+
+SupportedMassageTypes:
+  datatype: string[]
+  type: attribute
+  allowed:  ['ROLL','KNEAD','ANTI_FATIGUE','PULSE','STRETCH','LUMBAR','SHOULDER','WAVE','RELIEF','GLIDE']
```

Comment:
MoM:

- M: Do we think everyone would have the same pattern? Or ask the vehicle which pattern it has?
- E: An alternative would be to not standardize the list, but leave it to OEMs
- S: We had descriptions from technical and marketing perspective, and they differ, so complex to align

Reply:
MoM:

- E: We need to decide if we want to have the list of "allowed" value in the standard catalogue.
- D: We cannot model which  capabilities a vehicle has with current VSS.
- D: The terms actuator/sensor is misleading

Reply:
As it seems from the discussion that the list will not be "standardized" I suggest that we remove the `allowed` list and instead add a comment, possibly like:

`comment: OEMs may define a list of identifiers for supported massage types using the allowed keyword`

Reply:
MoM:

- Melissa: Agrees to remove
- Melissa: Do we need to formalize display name? With human readable name